server/src/main/scala/org/ensime/server/Server.scala

```diff
@@ -115,13 +117,23 @@ object Server extends AkkaBackCompat {
   Slf4jSetup.init()
 
   val log = LoggerFactory.getLogger("Server")
+  val XDG_CONFIG_HOME = " ~/.config/ensime-server"
+
+  lazy val shutDownOnDisconnect = Option(System.getProperty("ensime.explode.on.disconnect")).isDefined
```

Comment:
A bit confused here. You mean to fetch ``XDG_CONFIG_HOME``  value from ``reference.conf`` ? At the moment we dont have a ``reference.conf`` , does that have to be created ?